app/assets/stylesheets/layout/common/_header.scss

```diff
@@ -180,6 +180,24 @@
   }
 }
 
+.l-header__content {
+  position: fixed;
+  background: $color-green-primary;
+  z-index: 150;
```

Comment:
The value for this needs to be greater than that for the sticky newsletter signup, which is 100, so that would need to change as well. Might be better left out of this card and added as a more generic piece of work.